meilisearch-release-changelog/v1.6.0.md

```diff
@@ -0,0 +1,207 @@
+
+### Hybrid & semantic search
+
+You can perform a hybrid search by using the `hybrid` field when calling the `POST /index/:index_uid/search` route.
+
+Here is an example of a hybrid search payload:
+
+```json
+{
+    "q": "Plumbers and dinosaurs",
+    "hybrid": {
+        "semanticRatio": 0.9,
+        "embedder": "default"
+    }
+}
+```
+
+`embedder` is the embedder you choose to perform the search among the ones you defined in your settings.
+`semanticRatio`: the value should be between 0 and 1. The default value is 0,5. 1 corresponds to a full semantic search, whereas 0 is about a full-text search.
+
+### ⚠️ Breaking changes for beta users of the previous version of vector search
+
+For people who used Meilisearch with the experimental vector search feature (between v1.3.0 and v1.5.0), some changes happened in the API usage:
+
+- Before, sending your vectors without defining any models to use vector search was possible. Now, we have to define a model in the settings.
+
+```json
+"embedders": {
+    "default": {
+      "source": { "userProvided": { "dimensions": 512 } }
+    }
+}
+```
+
+- Meilisearch now supports multiple embedders, so the format for sending documents and vectors changed. Vectors are not arrays anymore but JSON objects.
+
+Before, in your document you provided:
+
+```json
+"_vectors": [
+  [0.0, 0.1]
+]
+```
+
+Now the format is:
+
+```json
+"_vectors": {
+  "image2text": [0.0, 0.1, ...]
+}
+```
+
+To know more about the new usage, refer to the sections above about settings or to [the documentation](TBD)
+
+### More technical information
+
+You can check out
+- [this article](https://blog.kerollmops.com/spotify-inspired-elevating-meilisearch-with-hybrid-search-and-rust)
+- [Arroy](https://github.com/meilisearch/arroy), the opensource repository based on [Annoy](https://github.com/spotify/annoy), written in Rust. This repository is created and maintained by the Meilisearch engine team. This is a Rust library to search for vectors in space that are close to a given query vector
+
+Done in #4226 by @dureuill, @irevoire, @Kerollmops and @ManyTheFish.
+
+## Improve indexing speed
+
+This version introduces huge indexing performance improvements. Meilisearch has been optimized to
+- store and pre-compute less data than in the previous versions
+- re-index and delete only the necessary data when updating a document. For instance, if updating only one field in the document, Meilisearch will only recompute this field and will no longer re-index the complete document.
+
+Some metrics: on an e-commerce dataset of 2.5Gb of documents, we noticed more than a 50% time reduction when adding documents for the first time. With a scenario updating the documents frequently and partially, the reduction is about 50% or even 75%. Most of all, the indexing time does not exponentially increase anymore.
```

Comment:
> The indexing time does not exponentially increase anymore.

It's a bit true and a bit false. Technically, it's not an exponential curve as _e<sup>n</sup>_, and the curve tends to be more linear than before. However, it's still kind of exponential because the indexing time of a constant amount of data continues to rise with the amount of data already indexed.
Let's say you index an infinite amount of tasks of the exact same size in Meilisearch, you may see an exponential curve at one point.
Below the blue curve could seem linear:
![Capture d’écran 2023-12-19 à 13 39 24](https://github.com/meilisearch/engine-team/assets/6482087/84264277-9e16-4d7f-80ee-fdd9c2417010)

But if we de-zoom, we see the exponential curve:
![Capture d’écran 2023-12-19 à 13 40 02](https://github.com/meilisearch/engine-team/assets/6482087/e6971798-cabd-4055-8341-a062a7e64fc8)

Sorry if I give too much detail, but I found it interesting to explain that we are not sure that Meilisearch is no longer exponential. 😊

Reply:
@ManyTheFish Do you agree if I only remove the mention of the "exponential" part? Otherwise, how would you explain it with other words?

Reply:
I think removing the exponential part is good; it doesn't remove any important information.
But, we could say that "the write amplification is (highly) reduced", or if we want to keep the "exponential" thing, we keep it in the context of a filterable attribute modification, because, in this context, there is close to 0 write amplification. The write amplification of Meilisearch comes mainly from the searchable attributes :)

Reply:
I'm ok with removing the mention of exponential!